eng/tools/generator/repo/sdk.go

```diff
@@ -204,3 +212,130 @@ func (s *sdkRepository) checkoutBack(ref *plumbing.Reference) error {
 	}
 	return s.Checkout(&opt)
 }
+
+var prBody = `
+<!--
+Thank you for contributing to the Azure SDK for Go.
+
+Please verify the following before submitting your PR, thank you!
+-->
+
+- [ ] The purpose of this PR is explained in this or a referenced issue.
+- [ ] The PR does not update generated files.
+   - These files are managed by the codegen framework at [Azure/autorest.go][].
+- [ ] Tests are included and/or updated for code changes.
+- [ ] Updates to [CHANGELOG.md][] are included.
+- [ ] MIT license headers are included in each file.
+
+[Azure/autorest.go]: https://github.com/Azure/autorest.go
+[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md
+`
+
+var confirmComment = "@%s Please confirm the SDK package change: %s."
+
+func CreatePullRequest(ctx context.Context, client *github.Client, owner, repo, fork, branch, body string) (*github.PullRequest, *github.Response, error) {
```

Comment:
These operations do not belong to SDK repository. You could create another sub-package for github helper.

Reply:
to repo/github.go file